easybuild/easyblocks/s/scipy.py

```diff
@@ -198,6 +198,18 @@ def install_step(self):
         """Custom install step for scipy: use ninja for scipy >= 1.9.0"""
         if self.use_meson:
             MesonNinja.install_step(self)
+
+            # copy PKG-INFO file included in scipy source tarball to scipy-<version>.egg-info in installation,
+            # so pip is aware of the scipy installation (required for 'pip list', 'pip check', etc.);
+            # see also https://github.com/easybuilders/easybuild-easyblocks/issues/2901
+            pkg_info = 'PKG-INFO'
+            target_egg_info = os.path.join(self.installdir, self.pylibdir, 'scipy-%s.egg-info' % self.version)
+            if os.path.isfile(pkg_info):
+                copy_file(pkg_info, target_egg_info)
+            else:
+                cwd = os.getcwd()
+                print_warning("%s not found in %s, so can't use it to create %s!", pkg_info, cwd, target_egg_info,
```

Comment:
Error out here? If this has failed you are going to have problems installing anything that uses `scipy`!

Reply:
Sanity check should catch the problem, but sure, I can make this a hard error

Reply:
OK, the sanity check catching this is fine.